db/compaction.cc

```diff
@@ -284,32 +284,32 @@ bool Compaction::KeyNotExistsBeyondOutputLevel(
   assert(input_version_ != nullptr);
   assert(level_ptrs != nullptr);
   assert(level_ptrs->size() == static_cast<size_t>(number_levels_));
-  assert(cfd_->ioptions()->compaction_style != kCompactionStyleFIFO);
```

Comment:
@sagar0 it gets called after we added compaction to FIFO: https://github.com/facebook/rocksdb/commit/264d3f540c05c22ce498b068083ea97096b459b7 . It was my mistake that I didn't remove this assert in the PR.